[2021-06-27 18:01:51,868] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_mysql_invoice 2021-06-27T18:01:27.549223+00:00 [queued]>
[2021-06-27 18:01:51,892] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_mysql_invoice 2021-06-27T18:01:27.549223+00:00 [queued]>
[2021-06-27 18:01:51,893] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 18:01:51,894] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-06-27 18:01:51,897] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 18:01:51,928] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): export_mysql_invoice> on 2021-06-27T18:01:27.549223+00:00
[2021-06-27 18:01:51,938] {standard_task_runner.py:52} INFO - Started process 7080 to run task
[2021-06-27 18:01:51,943] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgretomysql', 'export_mysql_invoice', '2021-06-27T18:01:27.549223+00:00', '--job-id', '89', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/postgresql-collect.py', '--cfg-path', '/tmp/tmpvauw3r_k', '--error-file', '/tmp/tmpelbur5my']
[2021-06-27 18:01:51,945] {standard_task_runner.py:77} INFO - Job 89: Subtask export_mysql_invoice
[2021-06-27 18:01:52,014] {logging_mixin.py:104} INFO - Running <TaskInstance: postgretomysql.export_mysql_invoice 2021-06-27T18:01:27.549223+00:00 [running]> on host b69deda3a989
[2021-06-27 18:01:52,111] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=tech@innospark.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgretomysql
AIRFLOW_CTX_TASK_ID=export_mysql_invoice
AIRFLOW_CTX_EXECUTION_DATE=2021-06-27T18:01:27.549223+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-27T18:01:27.549223+00:00
[2021-06-27 18:01:52,112] {logging_mixin.py:104} INFO - Entering export_mysql_to_csv docker-mysql
[2021-06-27 18:01:52,147] {base.py:78} INFO - Using connection to: id: docker-mysql. Host: 172.27.240.1, Port: 3306, Schema: , Login: root, Password: ***, extra: {'charset': 'utf8', 'local_infile': True}
[2021-06-27 18:01:52,201] {python.py:151} INFO - Done. Returned value was: None
[2021-06-27 18:01:52,223] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=postgretomysql, task_id=export_mysql_invoice, execution_date=20210627T180127, start_date=20210627T180151, end_date=20210627T180152
[2021-06-27 18:01:52,272] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-06-27 18:01:52,321] {local_task_job.py:151} INFO - Task exited with return code 0
